runtime/master/src/main/java/edu/snu/onyx/runtime/master/scheduler/SingleJobTaskGroupQueue.java

```diff
@@ -186,6 +192,26 @@ private boolean isSchedulable(final String candidateStageId, final String candid
     return true;
   }
 
+  private void print(final String calledMethod) {
```

Comment:
how about something like `log` instead of `print`, since what it does is it adds to the log.